connector/swagger/medco-connector.yml

```diff
@@ -914,6 +915,11 @@ definitions:
                   pattern: '^((\/[^\/]+)+\/)$'
             encrypted:
               type: "boolean"
+      patientSetIDItems:
```

Comment:
Linked to the other comment, since a patient set is just a type of item for i2b2, I would say to change this way of specifying the patient set ID, and put it along with the other query terms.
Also right now, it does not allow mixing together patient set IDs with other query terms (concepts or modifiers), which I believe is required (and enabled by i2b2).

I would suggest to:
- remove the array and make a simple field
- alternatively:
  1. put this as a field of `items` / `conceptItems` alongside `queryTerm`, `modifier`, etc.
  2. remove this field altogether and use the field `queryTerm` for specifying the patient set ID, and add a specific operator (e.g. `PATIENT_SET`) and/or add a `type` (e.g. `PATIENT_SET`)

Reply:
> Linked to the other comment, since a patient set is just a type of item for i2b2, I would say to change this way of specifying the patient set ID, and put it along with the other query terms.

Linked to my previous comment, this way we would lose the consistency check at the API level.

> Also right now, it does not allow mixing together patient set IDs with other query terms (concepts or modifiers), which I believe is required (and enabled by i2b2).

Actually it's possible, `conceptItems` and `patientSetIDItems `are both under `panel`